src/crimson/os/seastore/async_cleaner.cc

```diff
@@ -1454,11 +1442,13 @@ std::ostream &operator<<(std::ostream &os, AsyncCleaner::gc_stat_printer_t stats
          << ", dirty_tail_target=" << stats.cleaner->get_dirty_tail_target()
          << ", tail_limit=" << stats.cleaner->get_tail_limit();
     }
-    os << ", unavailable_unreclaimable="
-       << stats.cleaner->get_unavailable_unreclaimable_bytes() << "B"
-       << ", unavailable_reclaimble="
-       << stats.cleaner->get_unavailable_reclaimable_bytes() << "B"
-       << ", alive=" << stats.cleaner->stats.used_bytes << "B";
+    if (stats.cleaner->get_journal_type() == journal_type_t::SEGMENT_JOURNAL) {
```

Comment:
This can be dealt later after split the implementations, if it doesn't result in failures.

Reply:
I added this line because I encountered abort due to reclaim-related calculation.